### PR TITLE
header first touch

### PR DESCRIPTION
**Pull request by pemeraldy**

### CODE REVIEW

**Comment by hailelagi** [comment]
@pemeraldy Were you the first person to claim this story?

File: index.html [hunk 1]
```diff
@@ -1,39 +1,121 @@
...
-	<footer></footer>
-</body>
+      <section class="contact-me"></section>
+    </main>
 
-</html>
\ No newline at end of file
+    <footer></footer>
+
+    <script
```
Comment (hailelagi) [inline index.html]: Please put all your script tags in the header which will be fetched asynchronously. To understand _why_ this is the best approach read this [stack overflow question](https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup)

File: assets/css/portfolio.css [hunk 2]
```diff
@@ -1,3 +1,117 @@
-body{
-    background-color: #fff;
-}
\ No newline at end of file
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700&display=swap");
```
Comment (hailelagi) [inline assets/css/portfolio.css]: Please serve fonts locally.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (hailelagi) [inline .vscode/settings.json:2]: .gitignore IDE specific settings

File: assets/css/portfolio.css [hunk 4]
```diff
@@ -1,3 +1,117 @@
...
+  font-size: 14px;
+}
+header .navbar #collapsibleNavbar .nav-item.active {
+  border-top: 8px solid var(--leading-font-color);
+  color: var(--leading-font-color);
+}
+
+/* into content */
+.hero {
+  height: 90vh;
```
Comment (hailelagi) [inline assets/css/portfolio.css]: The page should fit on the screen and look good on 1440px by default. It does not. Please find a work around for this.

File: index.html [hunk 5]
```diff
@@ -1,39 +1,121 @@
...
+              >
+                <span class="navbar-toggler-icon"></span>
+              </button>
 
-		<section class="skills">
-		</section>
+              <!-- Navbar links -->
+              <div class="collapse navbar-collapse" id="collapsibleNavbar">
+                <ul class="navbar-nav ml-auto">
+                  <li class="nav-item active">
```
Comment (hailelagi) [inline index.html:65]: The active link highlight does not look like the design. Please take another look

**Review by hailelagi — CHANGES_REQUESTED** [review]
Very nice work! Man You've completed two tickets!!! There are some changes you need to make though. If someone doesn't beat you to it. I'll accept your work pending changes. Again good stuff!

File: index.html (hunk 1 above)
Comment (pemeraldy) [reply index.html]: Ok got it. On it.

**Pull request merged**